docs/source/notebooks/tpt.ipynb

```diff
@@ -0,0 +1,779 @@
+    "\n",
+    "$$\\mathbb{P}((i,j)) = 0\\;\\forall (i,j)\\in\\mathrm{Home}$$ \n",
+    "\n",
+    "and vice versa\n",
+    "\n",
+    "$$\\mathbb{P}((i,j)) = 1\\;\\forall (i,j)\\in\\mathrm{Bar}.$$\n",
+    "\n",
+    "The [law of total probability](https://en.wikipedia.org/wiki/Law_of_total_probability) tells us that to know the probability of an event $A$ with a known and at most countably infinite set of mutually exclusive events $\\{C_n : n\\in\\mathbb{N}\\}$ so that $\\sum_i \\mathbb{P}(C_i) = 1$, one can evaluate\n",
+    "\n",
+    "$$\n",
+    "\\mathbb{P}(A) = \\sum_n \\mathbb{P}(A\\mid C_n)\\mathbb{P}(C_n).\n",
+    "$$\n",
+    "\n",
+    "In our example and ignoring borders and barriers, this means that for a current state $s=(i,j)$ and $A=\\text{\"Home from state }s\\text{\"}$ the sample space consists of direct neighbor states, i.e.,\n",
+    "\n",
+    "$$\n",
+    "C = \\{ \\text{move from }s\\text{ to }s+(k,l) : (k,l)\\in \\{ -1, 0, 1 \\}^2 \\text{ and } (k,l)\\neq (0, 0) \\} \\}.\n",
+    "$$\n",
+    "\n",
+    "Then,\n",
+    "$$\n",
+    "\\mathbb{P}(\\text{Home from }s) = \\sum_{C_{(k,l)}\\in C} \\mathbb{P}(\\text{Home from }s+(k,l))\\mathbb{P}(C_{(k,l)}).\n",
+    "$$\n",
+    "\n",
+    "But the $\\mathbb{P}(C_{(k,l)})$ are exactly the transition probabilities to move from state $s$ to $s+(k,l)$, giving rise to the _forward committor_\n",
+    "\n",
+    "$$\n",
+    "q_s^{(+)} = q_{(i,j)}^{(+)} = \\sum_{(k,l)\\neq(0, 0)} q_{(i+k, j+l)}^{(+)}p_{(i,j),(i+k, j+l)},\n",
+    "$$\n",
+    "\n",
+    "where $p_{(i,j),(i+k, j+l)}$ is the probability to transition from state $s=(i,j)$ to one of its neighboring states.\n",
+    "\n",
+    "**More formally**, the forward committor probability is the probability to reach set of states $A$ before $B$. Using the first hitting time of a set $S$ given by\n",
+    "\n",
+    "$$T_{S}=\\inf\\{t \\geq 0 : X_t \\in S \\},$$\n",
+    "\n",
+    "it can be defined as\n",
+    "\n",
+    "$$ q^{(+)}_i=\\mathbb{P}_{i}(T_{A}<T_{B}). $$\n",
+    "\n",
+    "It also satisfies the boundary value problem\n",
+    "\n",
+    "$$\n",
+    "\\begin{aligned}\n",
+    "q_i^{(+)} &= 0 &&\\text{, for all }i\\in A,\\\\\n",
+    "q_i^{(+)} &= 1 &&\\text{, for all }i\\in B,\\\\\n",
+    "\\sum_j L_{ij} q^{(+)}_{j}&=0 &&\\text{, for all }i\\not\\in A\\cup B,\\\\\n",
+    "\\end{aligned}\n",
+    "$$\n",
+    "\n",
+    "where $L=P-\\mathbb{I}$ is the generator matrix and $P$ the transition matrix. The BVP-formulation is used to numerically find the forward committor.\n",
+    "\n",
+    "There is also the notion of a *backward committor*, which is the probability that given state $i$, the system has previously been in set $A$ rather than $B$. With detailed balance, the backward committor is given by $q_i^{(-)} = 1 - q_i^{(+)}$."
```

Comment:
can you comment on the plots? do we expect to see just the opposite colors or is it less straightforward?

Reply:
the MSM is reversible so it is expected with $p_i^{(+)} = 1 - p_i^{(-)}$

Reply:
thanks for adding the note!